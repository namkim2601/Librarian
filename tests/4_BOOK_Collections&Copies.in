SaVe CoLlEcTiOn colelctiondupes.csv
NuMbEr CoPiEs
AdD CoLlEcTiOn doesntexist.csv
add collection collection.csv
save collection collectionduplicates.csv
add collection collectionduplicates.csv
add collection collectioncopies.csv
number copies
exit
