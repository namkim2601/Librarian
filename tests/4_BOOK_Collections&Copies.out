user: No books in system.

user: No books in system.

user: No such collection.

user: 17 books successfully added.

user: Success.

user: No books have been added to the system.

user: 17 books successfully added.

user: Aeneid (Virgil): 2
Batman Volume 1 (Bob Kane): 2
Biggles in the Battle (W.E. Johns): 2
Halloween (Jamie Lee Curtis): 2
Harry Potter 1 (J.K. Rowling): 2
Harry Potter 2 (J.K. Rowling): 2
Harry Potter 3 (J.K. Rowling): 2
Harry Potter 4 (J.K. Rowling): 2
Harry Potter 5 (J.K. Rowling): 2
Harry Potter 6 (J.K. Rowling): 2
So Long and Thanks for all the Fish (Douglas Adams): 2
The Hitchhiker's Guide to the Galaxy (Douglas Adams): 2
The Hobbit (J.R.R. Tolkien): 2
The Iliad (Homer): 2
The Odyssey (Homer): 2
To Kill a Mockingbird (Harper Lee): 2
Wasteland (T.S. Eliot): 2

user: Ending Library process.
